src/routes/authentication-route.js

```diff
@@ -14,8 +15,19 @@ module.exports = function createAuthenticationRoute(repository) {
     return response.status(200).json({ token });
   }
 
+  async function comparePassword(request, response, next) {
+    const passwordDb = await repository.returnUser(request.body);
```

Comment:
You should not use names to confuse the reader. If you get the user back then the name of the variable should be `user`